update docs: improve tldr (quick start) (#192)
serokell/tezos-packaging: docs/baking.md

package, which provides predefined services for running baking instances on different
networks.

> First, add package repository:

Reviewer: With this change it almost looks like the `Update and install packages:` is only for the `On Raspberry Pi OS` due to the headings:
![Screenshot_20210507_151307](https://user-images.githubusercontent.com/12936619/117454864-e0dbba00-af46-11eb-8fd0-cac81d68da4f.png)

Now that there's a "section" that contains just the repository addition, I suggest going with:

    #### First, add the package repository:
    
    <a name="ubuntu"></a>
    On Ubuntu:
    ```
    # Add PPA with Tezos binaries
    sudo add-apt-repository ppa:serokell/tezos
    ```
    
    <a name="pios"></a>
    On Raspberry Pi OS:
    ```
    # Install software properties commons
    sudo apt-get install software-properties-common
    # Add PPA with Tezos binaries
    sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bionic main'
    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB
    ```
    
    #### Update and install packages:

instead.

package, which provides predefined services for running baking instances on different
networks.

To install them, run the following commands:
#### Add the package repository

On Ubuntu:

> <a name="ubuntu"></a>

Reviewer 2: This anchor and the anchor below were used in the `Quick Start` section

Author: fixed, replaced with link to anchor created from section header

2. update the `BAKER_ADDRESS_ALIAS` by editing the
    `/etc/default/tezos-baking-<network>` file.

> ## Quick Start

Reviewer: All LGTM now, however @itkach I recall you mentioning that this section could have been towards the top instead.

Alternatively it could be left where it is now, but instead add a link to "skip through" to it in the opening paragraph?

2. update the `BAKER_ADDRESS_ALIAS` by editing the
    `/etc/default/tezos-baking-<network>` file.

## Quick Start

<details>
 <summary>
   <em>Optional</em> Create new Ubuntu virtual machine...
 </summary>

A quick way to spin up a fresh Ubuntu virtual machine is
to use [Multipass](https://multipass.run/) (reduce disk if this is
to be used with a test network or a mainnet node in rolling history mode):

```
multipass launch --cpus 2 --disk 100G --mem 4G --name tezos
```

and then log in:

```
multipass shell tezos
```

> Note that on Windows and MacOS this VM will not have access to USB and
> thus is not suitable for using with Ledger Nano S.

</details>

1) Install `tezos-baking` package following either [Ubuntu](#ubuntu)
or [RaspberryPi OS](#pios) instructions.

2) Choose one of supported Tezos networks - `mainnet`, `florencenet` or
`edo2net` by setting environment variable:

```
tznet=florencenet
```

3) Initialize Tezos node from a snapshot and start node service:

```
snapshot_file=/tmp/tezos-$tznet.rolling

> snapshot_url=https://$tznet.xtz-shots.io/rolling
> wget $snapshot_url -O $snapshot_file

Reviewer: The `snapshot_url` variable seems unnecessary, I suggest avoiding it:
```suggestion
wget "https://$tznet.xtz-shots.io/rolling" -O $snapshot_file
```

Author: the variable is not strictly necessary, but I think it improves readability and usability by giving a meaningful name to otherwise nameless url, making wget command shorter, hinting that the url itself replacable/pluggable and giving an opportunity to set/change it independently of actual download command

Reviewer: Fair enough, those are good point :+1:

sudo -u tezos tezos-node-$tznet snapshot import $snapshot_file
sudo systemctl start tezos-node-$tznet
```

> If all you want is to run Tezos node, enable the service and stop
> here.

Reviewer: ```suggestion
If all you want is to run the Tezos node, enable the service and stop here:
```

```
sudo systemctl enable tezos-node-$tznet
```

<details>
 <summary>
  <em>Optional</em> Allow RPC access from virtual machine's host...
 </summary>

Update service configuration:

```
sudo systemctl edit tezos-node-$tznet
```

> An editor will open with service override configuration,

Reviewer: ```suggestion
An editor will open to override the service configuration.
```
I may be mistaken, but that sounds more like what it will do.

Reviewer 2: It'll indeed open an `override.conf` file, so I slightly prefer "An editor will open with service override configuration", maybe
"An editor will open service override configuration file" sounds a bit better

Reviewer: Oh I see, in this case the current wording sounds ok to me :+1:
